support/user-guide/amp-for-email.md

```diff
@@ -118,16 +118,21 @@ Example:
i5hbXBwcm9qZWN0Lm9yZy92MC9hbXAtc2VsZWN0b3ItMC4xLmpzIj48L3NjcmlwdD4KICAgIDxzY3JpcHQgYXN5bmMgY3VzdG9tLWVsZW1lbnQ9ImFtcC1mb3JtIiBzcmM9Imh0dHBzOi8vY2RuLmFtcHByb2plY3Qub3JnL3YwL2FtcC1mb3JtLTAuMS5qcyI+PC9zY3JpcHQ+CiAgICA8c3R5bGUgYW1wLWN1c3RvbT4KICAgICAgYm9keSB7CiAgICAgICAgZm9udC1mYW1pbHk6IFJvYm90bywgQXJpYWwsIHNhbnMtc2VyaWY7CiAgICAgICAgYmFja2dyb3VuZDogbGluZWFyLWdyYWRpZW50KC0zNmRlZywgI2YzYzA4NwogICAgICAgICAgLCAjZTg4YjIxKSBmaXhlZDsKICAgICAgfQogICAgICBoMSB7CiAgICAgICAgY29sb3I6ICNlODhiMjE7CiAgICAgIH0KICAgICAgLmNvbnRhaW5lciB7CiAgICAgICAgYm9yZGVyLXJhZGl1czogNXB4OwogICAgICAgIHRleHQtYWxpZ246IGNlbnRlcjsKICAgICAgICBwYWRkaW5nOiA1cHggMjBweCAyMHB4IDIwcHg7CiAgICAgICAgbWFyZ2luOiA0MHB4IGF1dG87CiAgICAgICAgd2lkdGg6IDUwMHB4OwogICAgICAgIG1heC13aWR0aDogOTAlOwogICAgICAgIGJhY2tncm91bmQ6IHdoaXRlOwogICAgICB9CiAgICAgIC5hbXAtZm9ybS1zdWJtaXQtc3VjY2VzcyAjZm9ybS1jb250ZW50IHsKICAgICAgICBkaXNwbGF5OiBub25lOwogICAgICB9CiAgICAgIGFtcC1zZWxlY3RvciBbb3B0aW9uXSB7CiAgICAgICAgbWFyZ2luOiAxMHB4OwogICAgICAgIHdpZHRoOiA0MCU7CiAgICAgICAgZGlzcGxheTogaW5saW5lLWJsb2NrOwogICAgICAgIGJvcmRlcjogMnB4IHNvbGlkICNlNmU2ZTY7CiAgICAgICAgYm9yZGVyLXJhZGl1czogNXB4OwogICAgICAgIHBhZGRpbmc6IDEwcHg7CiAgICAgICAgb3V0bGluZTogMDsKICAgICAgfQogICAgICBhbXAtc2VsZWN0b3IgW29wdGlvbl1bc2VsZWN0ZWRdIHsKICAgICAgICBib3JkZXItY29sb3I6ICNlODhiMjE7CiAgICAgICAgb3V0bGluZTogMDsKICAgICAgfQogICAgICAudXBkYXRlQnV0dG9uIHsKICAgICAgICBiYWNrZ3JvdW5kOiAjZTg4YjIxOwogICAgICAgIGNvbG9yOiB3aGl0ZTsKICAgICAgICBib3JkZXI6IDA7CiAgICAgICAgYm9yZGVyLXJhZGl1czogNXB4OwogICAgICAgIHBhZGRpbmc6IDEwcHggMjBweDsKICAgICAgICBmb250OiBpbmhlcml0OwogICAgICAgIGZvbnQtd2VpZ2h0OiAzMDA7CiAgICAgICAgbWFyZ2luLXRvcDogMTBweDsKICAgICAgfQogICAgPC9zdHlsZT4KICA8L2hlYWQ+CiAgPGJvZHk+CiAgICA8ZGl2IGNsYXNzPSJjb250YWluZXIiPgogICAgPCEtLSBhbXBieWV4YW1wZWwuY29tIGlzIHVzZWQgYXMgYSBwbGFjZWhvbGRlciB0aGF0IGFsbG93cyBmb3IgQU1QIHN1Ym1pdHMgLS0+CiAgICAgIDxmb3JtIGFjdGlvbi14aHI9Imh0dHBzOi8vYW1wYnlleGFtcGxlLmNvbS9zYW1wbGVzX3RlbXBsYXRlcy9wb2xsL3N1Ym1pdCIgbWV0aG9kPSJwb3N0Ij4KICAgICAgICA8ZGl2IGlkPSJmb3JtLWNvbnRlbnQiPgogICAgICAgICAgPGgxPlRoYW5rcyBmb3Igc3Vic2NyaWJpbmcgdG8gU3BhcmtQb3N0IEVtYWlscyE8L2gxPgogICAgICAgICAgPHA+V2hhdCBraW5kIG9mIGVtYWlscyBzaG91bGQgd2Ugc2VuZCB5b3U/PC9wPgogICAgICAgICAgPGFtcC1zZWxlY3RvciBuYW1lPSJzdWJzY3JpcHRpb25zIiBtdWx0aXBsZT4KICAgICAgICAgICAgPGRpdiBvcHRpb249ImFjY291bnQtbWV0cmljcyIgc2VsZWN0ZWQ+V2Vla2x5IEFjY291bnQgTWV0cmljczwvZGl2PgogICAgICAgICAgICA8ZGl2IG9wdGlvbj0iZGFpbHktcmV2aWV3cyIgc2VsZWN0ZWQ+RGFpbHkgRGVsaXZlcmFiaWxpdHkgUmVwb3J0PC9kaXY+CiAgICAgICAgICAgIDxkaXYgb3B0aW9uPSJuZXdzbGV0dGVyIiBzZWxlY3RlZD5CZXRhIFRlc3RlciBOZXdzbGV0dGVyPC9kaXY+CiAgICAgICAgICAgIDxkaXYgb3B0aW9uPSJwcm9kdWN0LXVwZGF0ZXMiIHNlbGVjdGVkPlByb2R1Y3QgVGVhbSBVcGRhdGVzPC9kaXY+CiAgICAgICAgICA8L2FtcC1zZWxlY3Rvcj4KICAgICAgICAgIDxpbnB1dCB0eXBlPSJzdWJtaXQiIHZhbHVlPSJVcGRhdGUiIGNsYXNzPSJ1cGRhdGVCdXR0b24iPgogICAgICAgIDwvZGl2PgogICAgICAgIDxkaXYgc3VibWl0LXN1Y2Nlc3M+CiAgICAgICAgICA8aDE+R290IGl0ISBXZSB3aWxsIHVwZGF0ZSB5b3VyIHByZWZlcmVuY2VzIHdpdGhpbiAyNCBob3VyczwvaDE+CiAgICAgICAgPC9kaXY+CiAgICAgIDwvZm9ybT4KICAgIDwvZGl2PgogIDwvYm9keT4KPC9odG1sPg==)
```

Comment:
There is an error in this example.  The form submit request fails.  We should probably move completely away from "https://ampbyexample.com".

<img width="1387" alt="Screen Shot 2021-01-08 at 2 17 29 PM" src="https://user-images.githubusercontent.com/1335605/104055161-45b31d80-51bc-11eb-9b81-350d656cb9c0.png">

Reply:
Yes, had not noticed that myself. Looks like that domain has been deprecated and content moved - see https://blog.amp.dev/2019/04/02/ampproject-org-moves-to-amp-dev/. I'll try to find out the new URL to use.

Reply:
This example seems close: https://amp.dev/documentation/components/amp-form/?format=email

Reply:
I can make the example work using
```
      <form action-xhr="/documentation/examples/api/submit-form-input-text-xhr" method="post">

```

This hits an endpoint on the amp.dev site

Reply:
> The relative URL '/documentation/examples/api/submit-form-input-text-xhr' for attribute 'action-xhr' in tag 'form' is disallowed.

It looks like the playground validation doesn't like the relative link, but prepending "https://amp.dev/" works!

<img width="1391" alt="Screen Shot 2021-01-13 at 11 26 53 PM" src="https://user-images.githubusercontent.com/1335605/104544878-de033500-55f6-11eb-89e4-e4de72ac9c0e.png">

Reply:
Good catch. Will change it.